packages/prisma/seed.ts

```diff
@@ -291,47 +291,47 @@ async function main() {
             recurringEventId: Buffer.from("yoga-class").toString("base64"),
             startTime: dayjs().add(1, "day").toDate(),
             endTime: dayjs().add(1, "day").add(30, "minutes").toDate(),
-            status: BookingStatus.PENDING,
+            status: BookingStatus.ACCEPTED,
```

Comment:
Yoga class does not require confirmation, so changing its status to ACCEPTED to be aligned to that.